### PR TITLE
Add func. test for Banner Error

### PR DESCRIPTION
**Pull request by leobcgov**
# Fixes or Implements [AB#10024](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/10024)

* [X] Enhancement
* [ ] Bug
* [ ] Documentation

## Description
- Functional Test for Banner Error.
image

If you are reviewing this PR, please adhere to the guidelines as [documented](https://github.com/bcgov/healthgateway/wiki/prguidance).

## Testing

* [ ] Unit Tests Updated
* [X] Functional Tests Updated
* [ ] Not Required

### Steps to Reproduce

If this is a bug, please provide details on how to reproduce the code.

### UI Changes

NO

### Browsers Tested

* [ ] Chrome
* [ ] Safari
* [ ] Edge
* [ ] Firefox

Provide an explanation for any test(s) not completed.

## Notes/Additional Comments

Any additional notes or comments that may be relevant.  Include any specialized deployment steps here.

### CODE REVIEW

File: Testing/functional/e2e/cypress/integration/timeline/errors.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+const { AuthMethod } = require("../../support/constants")
+
+describe('Banner Error', () => {
+    before(() => {
+        cy.intercept("GET", "/v1/api/Note/*", (req) => {
+            req.reply((res) => {
+                res.body.resultStatus = 0;
+                res.body['resultError'] = {
+                    resultMessage: "Error ABC",
+                    errorCode: "Error Code",
```
Comment (FuriousLlama) [inline Testing/functional/e2e/cypress/integration/timeline/errors.js:10]: The translator wont be able to put this in the correct format

Comment (leobcgov) [reply Testing/functional/e2e/cypress/integration/timeline/errors.js:10]: ahh right, let me change to test the translation as well

Comment (leobcgov) [reply Testing/functional/e2e/cypress/integration/timeline/errors.js]: updated!

**Review by FuriousLlama — APPROVED** [review]

File: Testing/functional/e2e/cypress/integration/timeline/errors.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+const { AuthMethod } = require("../../support/constants")
+
+describe('Banner Error', () => {
+    before(() => {
+        cy.intercept("GET", "/v1/api/Note/*", (req) => {
+            req.reply((res) => {
+                res.body.resultStatus = 0;
+                res.body['resultError'] = {
```
Comment (tiagograf-AOT) [inline Testing/functional/e2e/cypress/integration/timeline/errors.js:8]: lets use a fixture to return mock data

Comment (leobcgov) [reply Testing/functional/e2e/cypress/integration/timeline/errors.js:8]: as we dont have a lot of data to mock here, I don't think fixture is better. Personally, I prefer to see my test data included here as it is clearer, unless we have some good arguments for fixture vs. this?
Any thoughts @sslaws @FuriousLlama

Comment (FuriousLlama) [reply Testing/functional/e2e/cypress/integration/timeline/errors.js:8]: I like the idea that the mock data is stored in a file vs in code. +1 on using fixtures.

Comment (leobcgov) [reply Testing/functional/e2e/cypress/integration/timeline/errors.js]: mock data has been moved

**Review by tiagograf-AOT — APPROVED** [review]

**Pull request merged**